describealign.py

```diff
@@ -1110,33 +1110,55 @@ def combine_gui(video_files, audio_files, config_path):
       break
   combine_window.close()
 
-def migrate_config(old_path: Path, new_path: Path) -> None:
+def migrate_config(old_paths: list[Path], new_path: Path) -> bool:
   """
   Migrate configuration from old location.
   
-  Only runs if the old_path exists but new_path does not
+  Only runs if one of old_paths exists but new_path does not
+  
+  :param old_paths: A list of previous config file locations, in chronological order oldest to newest
+  :param new_path: The new config location
+  :returns: True if the config was migrated, else False
   """
-  if new_path.exists() or not old_path.exists():
-    return
+  if new_path.exists():
+    return False
   
-  old_data = old_path.read_text(encoding='utf-8')
-  new_path.write_text(old_data, encoding='utf-8')
-  print(f"Configuration migrated to {new_path}")
-  try:
-    old_path.unlink()
-  except OSError as exc:
-    print("Failed to remove old config:", *traceback.format_exception_only(exc))
-  else:
-    print("Successfully removed old config file.")
+  # Ensure the current path isn't in the old list
+  if new_path in old_paths:
+    old_paths.remove(new_path)
+  
+  # Work backwards in time, prioritize newest config locations
+  for old_path in reversed(old_paths):
+    if not old_path.exists():
+      continue
```

Comment:
Seems like running this twice would copy the 2nd newest over on the first run, then overwrite that with the 3rd oldest the second time.  Also not really happy with the whole concept here.  It made sense to me when there was just one old config file, as that could just be moved.  Don't think anyone's downloaded this since the last change and not convinced we need to preemptively handle future changes to the config directory.  Just seems like the bloat isn't worth it

Reply:
Because we `return True` at the end of the loop body, this will only run until it finds an old config, preferring the newest format. And due to the guard statement above (`if new_path.exists(): return False`), the function will only run if there is not a new config file. I can remove this if you think we only need to handle the one migration, but due to the guard statement this is a very quick return on all subsequent runs so I wouldn't consider it bloat.